packages/cli/src/commands/screenshot-diff/screenshot-diff.command.ts

```diff
@@ -1,18 +1,21 @@
 import { METICULOUS_LOGGER_NAME } from "@alwaysmeticulous/common";
 import { AxiosInstance } from "axios";
-import log from "loglevel";
-import { PNG } from "pngjs";
+import log, { Logger } from "loglevel";
 import { CommandModule } from "yargs";
 import { createClient } from "../../api/client";
 import { getDiffUrl, postScreenshotDiffStats } from "../../api/replay.api";
-import { CompareImageOptions, compareImages } from "../../image/diff.utils";
+import { CompareImageResult, compareImages } from "../../image/diff.utils";
 import {
   getOrFetchReplay,
   getOrFetchReplayArchive,
-  readReplayScreenshot,
+  getReplayDir,
+  getScreenshotFiles,
+  getScreenshotsDir,
 } from "../../local-data/replays";
 import { writeScreenshotDiff } from "../../local-data/screenshot-diffs";
 import { wrapHandler } from "../../utils/sentry.utils";
+import { readPng } from "../../image/io.utils";
+import { basename, join } from "path";
```

Comment:
Nitpick: run "optimize imports"

Reply:
My IDE thinks imports are ordered correctly here 🤔 wonder if should standardise import ordering with eslint / prettier?

Reply:
https://github.com/alwaysmeticulous/meticulous-sdk/pull/132/commits/eff308933321daf8e9a050292f84bddfc83d3f3c